include/varcache.h

```diff
@@ -15,13 +15,28 @@ struct VarCache {
 	struct PStr **var_list;
 };
 
+struct WelcomeVarLookup {
+	char *name;			/* key (string is WITHIN the structure) */
+	char *value;		/* value (string is WITHIN the structure) */
+	UT_hash_handle hh;	/* makes this structure hashable */
+};
```

Comment:
Instead of having this be "welcome vars". I think instead we should maybe support syncing of read-only parameters. More details in this comment, let's keep the discussion there because it impacts both PRs: https://github.com/pgbouncer/pgbouncer/pull/1179#discussion_r1881016656